src/containers/ModelPanel/index.tsx

```diff
@@ -100,7 +100,7 @@ class ModelPanel extends React.Component<ModelPanelProps, {}> {
             [VIEWER_ERROR]: isNetworkedFile ? (
                 <NetworkFileFailedText />
             ) : (
-                <NoTypeMappingText />
+                checkboxTree
```

Comment:
can remove line 43 too now (linter says so!).  It won't let me put the comment on that line because it's not part of the diff, but the linter warning is right there

Reply:
Oh thanks!!